src/db/seed.js

```diff
@@ -0,0 +1,59 @@
+// external imports
+const mysql = require("mysql2");
+const colors = require("colors");
+
+colors.setTheme({
+  greeting: ["rainbow"],
+  success: ["bgGreen", "black"],
+  warning: ["bgBrightYellow", "black", "bold"],
+  fail: ["bgRed", "white", "bold"],
+  message: ["bgBrightCyan", "black"],
+});
+
```

Comment:
This is great. I loved this. It's such a small thing but makes a world of difference, really great job.